src/App.js

```diff
@@ -1,28 +1,42 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
-import { BrowserRouter } from 'react-router-dom';
+import { 
+  BrowserRouter as Router,
+  Route,
+  Link
+} from 'react-router-dom';
+import OpenTrunk from "./open-trunk";
+import Books from "./books";
```

Comment:
How about appending the word Section? E.g. BookSection

Reply:
Is appending section necessary? Would books not be enough? I only mention this because this would make all of our compartment names much longer.

Reply:
true true